.circleci/config.yml

```diff
@@ -94,6 +142,64 @@ jobs:
             ./bin/linux/amd64/github-release release -t "${CIRCLE_TAG}"
             cat release-description.md | grep ${PROJECT} | ./bin/linux/amd64/github-release edit -t ${CIRCLE_TAG} -d -
 
+  py3to2:
+    docker:
+      - image: python:3.7.0-slim
+    steps:
+      - checkout
+      # Install some requirements
+      - run: pip install pytest==2.3.1 future==0.17.1 strip-hints==0.1.7 future-fstrings[rewrite]==1.2.0
```

Comment:
If we're pinning a bunch of stuff, let's put this in a `requirements.txt` for easier management. Maybe `requirements.py3to3.txt` in the root of the repo, or something like that?